lib/oban/job.ex

```diff
@@ -112,6 +112,7 @@ defmodule Oban.Job do
     field :conf, :map, virtual: true
     field :replace, {:array, :any}, virtual: true
     field :unique, :map, virtual: true
+    field :duplicate?, :boolean, virtual: true, default: false
```

Comment:
How about `conflict?` instead? That is less ambiguous than `duplicate?` when combined with the `replace` option.